Components/Text-Utilities/text.css

```diff
@@ -0,0 +1,14 @@
+.small-text {
+  font-size: 0.6rem;
+}
+
+.gray-text {
+  color: gray;

```

Comment:
Everything looks good but you can give variable for color

Reply:
Thanks for the suggestion, I have variables defined for colours in default.css, but given that "gray" is a color having a predefined proper name in CSS, I think due to that no reason to use a CSS variable in this case.